client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -30,9 +35,11 @@ export const fromApi = response => {
 		throw new Error( 'missing settings' );
 	}
 
-	return response.data;
+	return normalizeSettings( response.data );
 };
 
+export const toApi = settings => filterSettingsByActiveModules( sanitizeSettings( settings ) );
```

Comment:
If we don't plan to add any tests to `toApi`, maybe we shouldn't export it at all?

My personal opinion is that 1-2 simple tests would be helpful here in case we decide to alter something in the future.